Doc/includes/email-read-alternative.py

```diff
@@ -9,6 +9,9 @@
 from email.parser import BytesParser
 
 # An imaginary module that would make this work and be safe.
+# The magic_html_parser has to rewrite the href="cid:...." attributes to
+# point to the filenames in partfiles.  It also has to do a safety-sanitize
+# of the html.  It could be written using html.parser.
 from imaginary import magic_html_parser
```

Comment:
We could also rename imaginary to imaginary_module and then it won't look like a real module name.